src/main/resources/site/parts/activeStatistics/activeStatistics.ts

```diff
@@ -0,0 +1,143 @@
+import { Content } from 'enonic-types/content'
+import { Request, Response } from 'enonic-types/controller'
+import { ResourceKey, ThymeleafLibrary } from 'enonic-types/thymeleaf'
+import { React4xp, React4xpObject, React4xpResponse } from '../../../lib/types/react4xp'
+import { ActiveStatisticsPartConfig } from './activeStatistics-part-config'
+import { Statistics } from '../../content-types/statistics/statistics'
+import { CmsStatistic, XpStatistic } from "../../../lib/types/relatedStatistics";
+
+const {
+  data: {
+    forceArray
+  }
+} = __non_webpack_require__('/lib/util')
+const {
+  get
+} = __non_webpack_require__('/lib/xp/content')
+const {
+  getContent,
+  getComponent,
+  pageUrl
+} = __non_webpack_require__('/lib/xp/portal')
+const {
+  render
+} = __non_webpack_require__('/lib/thymeleaf')
+const {
+  renderError
+} = __non_webpack_require__('/lib/ssb/error/error')
+const {
+  hasPath
+} = __non_webpack_require__('/lib/vendor/ramda')
+const {
+  localize
+} = __non_webpack_require__('/lib/xp/i18n')
+
+const React4xp: React4xp = __non_webpack_require__('/lib/enonic/react4xp')
+const view: ResourceKey = resolve('./activeStatistics.html')
+
+exports.get = (req: Request): React4xpResponse | Response => {
+  try {
+    return renderPart(req)
+  } catch (e) {
+    return renderError(req, 'Error in part ', e)
+  }
+}
+
+exports.preview = (req: Request): React4xpResponse => renderPart(req)
+
+function renderPart(req: Request): React4xpResponse {
+  const page: Content = getContent()
+  const part: ActiveStatisticsPartConfig = getComponent().config
+  const activeStatistics: Array<CmsStatistic|XpStatistic> = part.relatedStatisticsOptions ? forceArray(part.relatedStatisticsOptions) : []
```

Comment:
Suggestion: instead of "Array<CmsStatistic|XpStatistic>" --> "ActiveStatisticConfig['relatedStatisticsOptions']". That way we can use the automatically generated types as opposed to making a new manually.